components/src/components/convert/convert.tsx

```diff
@@ -159,18 +159,31 @@ export class Convert {
     }
 
     // The CLI tool that corresponds with the language we're converting from.
-    private get conversionTool(): { name: string; githubURL?: string } {
+    private get conversionTool(): { name: string; githubURL?: string, issueURL?: string } {
         switch (this.from) {
             case "tf":
+                return {
+                    name: "tf2pulumi",
+                    githubURL: "https://github.com/pulumi/tf2pulumi",
+                    issueURL: "https://github.com/pulumi/tf2pulumi/issues",
+                };
             case "kube":
+                return {
+                    name: "kube2pulumi",
+                    githubURL: "https://github.com/pulumi/kube2pulumi",
+                    issueURL: "https://github.com/pulumi/kube2pulumi/issues",
+                };
+            case "arm":
+                return {
+                    name: "arm2pulumi",
+                    githubURL: "https://github.com/pulumi/arm2pulumi",
+                    issueURL: "https://github.com/pulumi/pulumi-azure-nextgen/issues?q=is%3Aissue+is%3Aopen+label%3Aarm2pulumi",
```

Comment:
I just created the arm2pulumi repo and moved open issues over. This should now be https://github.com/pulumi/arm2pulumi/issues